pages/index.tsx

```diff
@@ -27,18 +28,28 @@ export default function Home() {
   const [packageNamespaces, setPackageNamespaces] = useState(initialPackageNamespaces)
   const [packageNames, setPackageNames] = useState(initialPackageNamespaces)
   const [packageVersions, setPackageVersions] = useState(initialPackageNamespaces)
+  const [renderedInitialGraph, setRenderedInitialGraph] = useState(false)
+
+  // query params - this is used to pass in node id's to go get
+  // test with nodeId: 3597773
```

Comment:
This nodeid is only valid during your import of data. If you rerun ingestion it should provide a different set of node ids

Reply:
yeah I just forgot to delete this.  fixing